### ISSUE TITLE
CircleCI "Tests" tab support (xml file)? 

### ISSUE THREAD

**Issue by Benjamin8585** [adamgruber/mochawesome#298]
**Is your feature request related to a problem? Please describe.**

On **CircleCI** there is a `Tests` tab where you can see test results and you can setup it only with a `xml` file. `mocha-junit-reporter` for example does this kind of `xml` generation but unfortunatly I can't use two reporters at the same time.

**Describe the solution you'd like**

Is there a way you can provide an option to integrate `xml` file in results? For example checking if `xml` option is here and just run `mocha-junit-reporter` directly from your library?

Thanks for your great library

**Comment by danhlee329** [comment]
There are options in Cypress to be able to [do this outside of mochawesome](https://docs.cypress.io/guides/tooling/reporters#Multiple-reporters)

**Comment by mikhail-g** [comment]
It would be nice to have `xml` in the reporter config so if set to true it would create junit report too reusing the context set for mochawesome. This way you'd need to configure screenshots and other additional info only once

**Comment by adamgruber** [comment]
mochawesome produces its own HTML/JSON output and isn't a JUnit/XML reporter, so this is out of scope for it directly. To feed CircleCI's Tests tab alongside the mochawesome report, run multiple reporters at the mocha/Cypress layer — e.g. [`mocha-multi-reporters`](https://www.npmjs.com/package/mocha-multi-reporters) (or Cypress's [multiple reporters](https://docs.cypress.io/guides/tooling/reporters#Multiple-reporters)) with `mocha-junit-reporter`. Closing as out of scope.

---
_Triaged and posted via Claude Code, on behalf of @adamgruber._
